File: app/src/main/java/com/vaibhav/moviedetails/listeners/MovieListener.java
package com.vaibhav.moviedetails.listeners;

import com.vaibhav.moviedetails.data.Movie;

/**
 * @author Vaibhav Bhandula on 09/10/18.
 */
public interface MovieListener {

    void onLikeMovie(Movie movie);

    void onUnLikeMovie(Movie movie);
}

File: app/src/main/java/com/vaibhav/moviedetails/view/listener/MovieListener.kt
package com.vaibhav.moviedetails.view.listener

import com.vaibhav.moviedetails.data.Movie

/**
 * @author Vaibhav Bhandula on 09/04/19.
 */
interface MovieListener {
    fun onLikeMovie(movie: Movie)
    fun onUnLikeMovie(movie: Movie)
}
